build.cake

```diff
@@ -248,6 +248,24 @@ Task("Clean")
     CleanDirectories("./**/bin", (fsi)=> !fsi.Path.FullPath.StartsWith("tools"));
 });
 
+Task("provision-net6sdk")
```

Comment:
Ideally we can pull this from a feed somewhere?

@jonathanpeppers  @Redth ?

Reply:
I think you can pull the Xamarin.iOS / Xamarin.Android builds from github statuses if you want:

![image](https://user-images.githubusercontent.com/840039/104856449-5bb99000-58d8-11eb-8ded-490812d5ae9e.png)
![image](https://user-images.githubusercontent.com/840039/104856459-6d029c80-58d8-11eb-8c4f-2ed44211dd54.png)

I don't know if I would try get the a newer .NET 6 builds. Our installer sets a flag file to enable "workloads" dependent on the version of .NET 6. It also breaks things occasionally.